modules/conversion-center/module.php

```diff
@@ -96,6 +105,50 @@ public function __construct() {
 		add_action( 'wp_ajax_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 		add_action( 'wp_ajax_nopriv_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 
+		if ( ! static::is_pro_active() ) {
+			add_action( 'wp_footer', function () {
+				$query = new \WP_Query( [
+					'post_type' => ConversionCenterModule::CPT_CONTACT_PAGES,
+					'posts_per_page' => -1,
```

Comment:
Why not just `1` for `posts_per_page`?

Reply:
@louiswol94 this is to handle an edge case, which is that the user has pro installed, has set various display conditions, and then removes pro. in that case I want to only show the instance that was set to display and the whole site